tests/testthat/test-pkg-dplyr.R

```diff
@@ -351,3 +351,23 @@ test_that("dplyr mutating joins for substrait_compiler support `by`, `keep`, and
     c("number_x", "letter", "number_y", "LETTER")
   )
 })
+
+test_that("distinct.SubstraitCompiler works", {
+  skip_if_not_installed("dplyr")
+
+  compiler <- substrait_compiler(data.frame(a = c(1, 1, 2, 2, 3)))
+  out <- distinct(compiler)
+  plan <- out$plan()
+  expect_s3_class(plan$relations[[1]]$root$input$aggregate, "substrait_AggregateRel")
+  #TODO: work out what to test here
```

Comment:
Bit stuck on what to test here, as I'm wanting to test results not relations, but it doesn't fit in with other tests

Reply:
Thanks - done now!